uds/uds_config_tool/odx/param.py

```diff
@@ -0,0 +1,62 @@
+from typing import List
+
+from uds.uds_config_tool import DecodeFunctions
+from uds.uds_config_tool.odx.diag_coded_types import (DiagCodedType,
+                                                      MinMaxLengthType)
+
+
+class Param():
+    """represents a PARAM with its DIAG CODED TYPE, name, byte position and (optionally) data
+
+    data is set when parsing a uds response and can then be decoded
+    """
+
+    def __init__(self, shortName: str, bytePosition: int, diagCodedType: DiagCodedType, data=None) -> None:
+        """initialize attributes
+
+        :param shortName: name defined in ODX as str
+        :param bytePosition: byte position in the response of the param
+        :param diagCodedType: the Param's DiagCodedType containing length and decode info
+        :param data: the part of the uds response that belongs to this Param
+        """
+        self.shortName = shortName
+        self.bytePosition = bytePosition
+        self.diagCodedType = diagCodedType
+        self.data = data
+
+    def calculateLength(self, response: List[int]) -> int:
```

Comment:
We should consider making all of the package PEP8-compliant at some point (general method/variable naming should be snake case)

Reply:
fixed